File: testdata/bench/go/BenchmarkTest00091.go
package bench

import (
	"net/http"
	"text/template"
)

// Fixed template, user data in context only
func Handler00091(w http.ResponseWriter, r *http.Request) {
	name := r.FormValue("name")
	t, _ := template.New("t").Parse("Hello, {{.Name}}!")
	t.Execute(w, map[string]string{"Name": name})
}

File: testdata/bench/kotlin/BenchmarkTest00091.kt
package bench

import io.ktor.server.application.*
import io.ktor.server.request.*
import io.ktor.server.response.*
import com.google.gson.Gson

// Safe: Gson deserialization to typed class
data class UserRequest91(val name: String, val age: Int)

suspend fun handler00091(call: ApplicationCall) {
    val body = call.receiveText()
    val user = Gson().fromJson(body, UserRequest91::class.java)
    call.respondText("Hello ${user.name}")
}
